pep-0646.rst

```diff
@@ -446,8 +482,8 @@ hinder them when migrating a legacy code base to use ``TypeVarTuple``.
 Multiple Unpackings in a Tuple: Not Allowed
 '''''''''''''''''''''''''''''''''''''''''''
 
-As with ``TypeVarTuples``, `only one <Multiple Type Variable Tuples:
-Not Allowed_>`_ unpacking may appear in a tuple:
```

Comment:
And here I'd clarify:

"only one unpacking may appear in a tuple or in a variadic generic type"